doc/quickstart.md

```diff
@@ -0,0 +1,208 @@
+Welcome to
+# Pyccel
+
+ ## What is Pyccel
+
+-   static compiler for Python 3, using Fortran or C as backend language.
+-   started as small open-source project in 2018 at IPP Garching.
+-   public repository is now hosted on GitHub, freely available for download.
+
+ ## Python’s objects, variables, and garbage collection
+
+ Python is an **interpreter** language, **dynamically typed** and **garbage-collected**.
+
+ ### Python object
+
+-   is created by the Python interpreter when `object.__new__()` is invoked (e.g. as a result of an expression).
+-   can be either mutable or immutable, but its type never changes.
+-   resides in memory and has a **reference count**.
+-   is accessed through one or more Python variables.
+-   is destroyed by the garbage collector when its reference count drops to zero.
+
+For more details about Python object, see [this](https://docs.python.org/3/tutorial/classes.html).
+
+ ### Python variable
+
+-   is a reference to a Python object in memory.
+
+-   is created with an assignment operation `x = expr`:
+    -   if the variable `x` already exists, the interpreter reduces the reference count of its object
+    -   a new variable `x` is created, which references the value of expr.
+
+-   can be destroyed with the command del `x`.
+
+For more details about Python variable, see [this](https://www.w3schools.com/python/python_variables.asp).
+
+ ## Static typed languages
+	
```

Comment:
can you point the other differences between a statically-typed language and python 
so the user know why we need to add somethings like (garbage collector, and some error that we raise)